src/shipit_uplift/shipit_uplift/api.yml

```diff
@@ -154,6 +154,19 @@ paths:
         200:
           description: Code coverage information by directory
 
+  /coverage_by_changeset/{changeset}:
```

Comment:
I recommend using a prefix of '/coverage/' for all coverage end points.

Reply:
I will do this in a follow-up.

Reply:
Actually, I've changed my mind and I've done it.